### ISSUE TITLE
[BUG_SP] Course generator can't fit more headlands

### ISSUE THREAD

**Issue by Markus33-ITA** [Courseplay/Courseplay_FS22#472]
**Steps to reproduce**
I was creating the course for the modded 11 meter tmc self-propelled forage harvester and the bug appeared. and it also happened a few days ago.

**Actual behavior**
in a nutshell, the mouse allows you to explore the map with the mod menu open only with movement without making any clicks

**Screenshots**
![fsScreen_2022_01_10_17_54_06](https://user-images.githubusercontent.com/96330304/148807050-92bfbc6a-f0b7-4c3f-8a4a-3fff607d74a3.png)

**Courseplay version**
Version used: 7.0.0.17

**Log**
the bug is at the bottom
[https://pastebin.com/dstnLQFU](https://pastebin.com/dstnLQFU)

**Map**
Map US

**Comment by Tensuko** [comment]
You should Update your game:
Version: 1.2.0.0 b14654

And is that modded harvester on modhub ?

**Comment by Markus33-ITA** [comment]
having a very low bandwidth connection, I tend to wait a bit before updating. tomorrow I'll download the upload.
no on farmingsimulator22mods. I have to see if it is well on the modhub.  ... on the modhub it doesn't fit. it is called FS22_TMCCancela_TJP1100
the bug had recurred once also with the john deere combine a few days ago.

**Comment by Tensuko** [comment]
Anything not from the ModHub will be ignored.
FarmingSimulator22mods is a freaking trash can full of stolen mods.

**Comment by Markus33-ITA** [comment]
you are right. sometimes when you need a major mod, you download it where it happens. I know well that the modder loses us. I will be more careful in the future and will search on better sites first.
I have seen and is also on modhoster. even if the hash is the same, as soon as I finish using the disc harrow, I do a test with the one downloaded from modhoster. and tomorrow I'll download the update

**Comment by pvaiko** [comment]
@Markus33-ITA have you modified the field?

**Comment by Tensuko** [comment]
Should be fixed in 7.0.0.18
Reopen if Issue persists.

**Comment by Markus33-ITA** [comment]
I've been messed up. in the meantime I had made several tests.
I hadn't made any changes in that field the only modified fields I have are 48 and 49 that I joined them.
I had tried with the update and v7.0.0.17 and it didn't happen to me and didn't even know how to replicate the bug. after more than 8 tests and after several bailouts, I gave up.
tonight I try with v7.0.0.18 and if it reappears I'll let you know.
as said, i don't know how to replicate it :(. knowing it would be much easier.
Thanks  for the speed in solving the bugs
